packages/fxa-content-server/app/scripts/templates/index.mustache

```diff
@@ -45,6 +45,9 @@
     {{#isInThirdPartyAuthExperiment}}
       {{^isSync}}
         {{{ unsafeThirdPartyAuthHTML }}}
+        <div id="tos-pp" class="text-grey-500 my-5 text-xs">
```

Comment:
This looks great for signin:
<img width="684" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/3b946c17-edf5-43bd-a8f5-62da42f91fe6">

Just a nit, index/signup has some extra space:
<img width="666" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/2fe6aca9-689e-488a-bda9-d77583f7ba1a">

I think we may want `text-grey-500 my-5 text-xs` here:
<img width="671" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/a9762f8c-da9f-4e63-91a7-5bfcb7e3cbfa">